governance/program/tests/use_veto_vote.rs

```diff
@@ -962,3 +962,69 @@ async fn test_veto_vote_with_community_max_voter_weight_addin_and_veto_not_tippe
 
     assert_eq!(proposal_account.state, ProposalState::Voting);
 }
+
+#[tokio::test]
```

Comment:
missing tests for full coverage:
cast electoral fails in cool off
relinquish succeeds in cool off
finalize fails in cool off

Reply:
The first 2 tests where there before and I added the 3rd 
1) cast electoral fails in cool off  https://github.com/solana-labs/solana-program-library/blob/ef697fedddda09c1202fe864fc18e31b9d88c4fb/governance/program/tests/process_cast_vote.rs#L1555
2) relinquish succeeds in cool off - it tests you can relinquish your `Yes` vote and then vote `No`  https://github.com/solana-labs/solana-program-library/blob/ef697fedddda09c1202fe864fc18e31b9d88c4fb/governance/program/tests/process_relinquish_vote.rs#L603
3) finalize fails in cool off https://github.com/solana-labs/solana-program-library/blob/1e61b65c68c87cc32b15409274575d7ed52b5ca7/governance/program/tests/process_finalize_vote.rs#L447